tests/cleaning_test.py

```diff
@@ -0,0 +1,161 @@
+from dfm.cleaning import QualityFilter
+
+texts = [
+    "jeg er glad",
+    "56789okd23456789098765sds",
+    "jeg er glad...",
+    "jeg er glad…",
+    "67 54 13 B7",
+    "#yolo # test ##",
+]
+
+
+texts.append(
+    """
+Helt normal tekst:
+Første vindstød af stærk storm - andre steder i landet ramt
+Frederikshavn blev det første sted, der mærkede vindstød af stormstyrke,
+og nu er det også det første sted, der har mærket vindstød af stærk storm. Hvis det er noget at prale af.
+
+Der er nemlig målt vindstød på 29,3 meter i sekundet. Det er stærk storm,
+når det er over 28,5 meter i sekundet.
+
+Andre dele af landet har nu også mærket de første vindstød af stormstyrke. 
+
+Odense Lufthavn har haft 24,5 meter i sekundet, mens Grønlandshavnen i Aalborg har ramt 24,7
+meter i sekundet. Det er mest centrale sted i landet, hvor der indtil videre er målet stormstyrke.
+"""
+)
+
+texts.append(
+    """
+[summary]
+
+- test 1
+- test 1
+    - test 2
+    - test 3
+        - test 1
+"""
+)
+
+texts.append(
+    """
+[summary]
+
+* test 1
+    * test 2
+    * test 3
+* test 1
+    * test 2
+        * test 3
+"""
+)
+
+
+def test_stop_word():
+    stop_words = set(
+        [
+            "er",
+            "jeg",
+            "det",
+            "du",
+            "ikke",
+            "at",
+            "en",
+            "og",
+            "har",
+            "vi",
+            "til",
+            "på",
+            "hvad",
+            "mig",
+            "med",
+            "de",
+            "for",
+            "den",
+            "så",
+            "der",
+            "dig",
+            "han",
+            "kan",
+            "af",
+        ]
+    )
+    qfilter = QualityFilter()
+    assert qfilter.stop_word(qfilter.nlp(texts[0]), n=2, stop_words=stop_words) is True
+    assert qfilter.stop_word(qfilter.nlp(texts[1]), n=1, stop_words=stop_words) is False
+
+
+def test_line_bullets_or_ellipsis():
+    qfilter = QualityFilter()
+    # bullet
+    assert (
+        qfilter.line_bullets_or_ellipsis(
+            qfilter.nlp(texts[1]), max_p_bullets=0.5, max_p_ellipsis=1
+        )
+        is True
```

Comment:
Don't really need all these "is True" statements, that's automatic in Python 🙂

Reply:
Not quite, `is True` is more restrictive (which I would argue is relevant in this case):
```
>>> assert "test"
>>> assert "test" is True
AssertionError
```

Reply:
Fair enough 👍